packages/css-processor/src/CSSNativeParseRun.ts

```diff
@@ -42,19 +42,8 @@ const extraLongViewStyles: Record<ExtraNativeLongViewStyleKeys, 'block'> = {
   shadowRadius: 'block',
   tintColor: 'block',
   transformMatrix: 'block',
-  translateX: 'block',
-  translateY: 'block',
-  borderCurve: 'block',
-  gap: 'block',
-  inset: 'block',
-  insetBlock: 'block',
-  insetInline: 'block',
-  marginBlock: 'block',
-  marginInline: 'block',
-  paddingBlock: 'block',
-  paddingInline: 'block',
-  objectFit: 'block'
-};
+  translateX: 'block'
+} as never;
```

Comment:
I don't think `as never` is an acceptable tradeoff; I would rather not lose type safety here.

Reply:
I adjusted the type settings - and I'm already implementing the support for missing style attributes, I thought that it would be benefitial for me to learn how it is implemented